### PR TITLE
[1LP][RFR] web_ui provisioning form removal

### PR DESCRIPTION
**Pull request by izapolsk**
This form is still used widely though we already have WT provisioning form. 
This PR intends to remove that web_ui form.

{{pytest: --long-running cfme/tests/infrastructure/test_provisioning_dialog.py cfme/tests/infrastructure/test_vm_migrate.py cfme/tests/infrastructure/test_vm_clone.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit b73d41cc660918d99cf8962840c5894f4e2a1ddb:

`cfme/common/vm_views.py`:
- :bangbang: Line 222:44: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 266:44: **E251** *unexpected spaces around keyword / parameter equals*

`cfme/tests/infrastructure/test_provisioning_dialog.py`:
- :bangbang: Line 62:25: **E201** *whitespace after '{'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 24c79b3d0feed157c13735680cf7d5c2824b3f2f:

`cfme/common/vm_views.py`:
- :bangbang: Line 222:44: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 266:44: **E251** *unexpected spaces around keyword / parameter equals*

`cfme/infrastructure/virtual_machines.py`:
- :bangbang: Line 687:71: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by izapolsk** [comment]
it turned out that @lkhomenk has a task to get rid of old provisioning form. So, I'm rely on him to replace all the rest old provisioning form usages

**Comment by izapolsk** [comment]
PRT is completely red due to collection changes. PR #5577 should fix that issue.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by izapolsk** [comment]
PRT looks good. 3 failed tests aren't related to my changes

File: widgetastic_manageiq/__init__.py [hunk 1]
```diff
@@ -1831,9 +1833,8 @@ def selected(self):
             return names[0]
 
     def select(self, name):
-        button = self._get_button(name)
         if self.selected != name:
-            button.click()
+            self.browser.element('.//input[@type="radio"]', parent=self._get_button(name)).click()
```
Comment (mfalesni) [inline widgetastic_manageiq/__init__.py]: **REQUIRED**: `self.browser.click('.//input[@type="radio"]', parent=self._get_button(name))` so you don't bypass the WT facilities.

File: widgetastic_manageiq/__init__.py [hunk 2]
```diff
@@ -1821,8 +1822,9 @@ def button_names(self):
     def selected(self):
         names = self.button_names
         for name in names:
-            if 'ng-valid-parse' in self.browser.classes('.//input[@type="radio"]',
-                                                        parent=self._get_button(name)):
+            bttn = self.browser.element('.//input[@type="radio"]', parent=self._get_button(name))
+            if ('ng-valid-parse' in self.browser.classes(bttn) or
+               bttn.get_attribute('checked') is not None):
```
Comment (mfalesni) [inline widgetastic_manageiq/__init__.py]: **REQUIRED**: Use double indent in here instead of indenting by 3 spaces.

Comment (izapolsk) [reply widgetastic_manageiq/__init__.py]: done

File: cfme/provisioning.py [hunk 3]
```diff
@@ -1,167 +1,9 @@
...
-submit_button = form_buttons.FormButton("Submit")
-
-
-def select_security_group(sg):
-    """TODO: Not even sure this is needed any more, but removal of it is not part of this PR"""
-    sel.wait_for_ajax()
-    sel.sleep(1)
-
-
-# TODO remove old form once all importers have moved to widget form
```
Comment (mshriver) [inline cfme/provisioning.py]: :tada:

File: cfme/tests/infrastructure/test_provisioning_dialog.py [hunk 4]
```diff
@@ -80,9 +79,7 @@ def provisioner(appliance, request, setup_provider, provider, vm_name):
     def _provisioner(template, provisioning_data, delayed=None):
         vm = Vm(name=vm_name, provider=provider, template_name=template)
         view = navigate_to(vm, 'Provision')
-        view.form.before_fill(provisioning_data)
```
Comment (mshriver) [inline cfme/tests/infrastructure/test_provisioning_dialog.py]: oh man this is ugly/bad, so good to see it go.

**Pull request merged**